.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - name: Set up NodeJS
         uses: actions/setup-node@v2-beta
         with:
-          node-version: 15.3.0
+          node-version: 20.11.1
```

Comment:
The updated linting packages were throwing errors with old node version; Node 15 is long unsupported, anyway.